src/pybind/mgr/orchestrator/module.py

```diff
@@ -295,7 +295,7 @@ def _list_devices(self, host=None, format='plain', refresh=False):
     @_cli_read_command(
         'orch ps',
         "name=host,type=CephString,req=false "
-        "name=daemon_type,type=CephChoices,strings=mon|mgr|osd|mds|iscsi|nfs|rgw|rbd-mirror,req=false "
+        "name=daemon_type,type=CephString,req=false "
```

Comment:
```suggestion
        "name=daemon_type,type=CephChoices,strings=mon|mgr|osd|mds|iscsi|nfs|rgw|rbd-mirror|node_exporter,req=false "
```

Reply:
@liewegas what's your take on this?

Reply:
the CephChoices is just tedious to maintain and tends to get out of sync (it appears in a several different commands)